src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import java.io.*;
+import java.util.ArrayList;
+
+public class Storage {
+  String path;
+
+  public Storage(String path) {
+    this.path = path;
+  }
+
+  public void saveBaby(ArrayList<Task> taskList) throws IOException {
```

Comment:
Interesting function name :) Maybe a JavaDoc note for those who might really not get it would be good as well!